ConsoleGUI/Controls/ListView.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using ConsoleGUI.Data;
+using ConsoleGUI.Input;
+using ConsoleGUI.UserDefined;
+
+namespace ConsoleGUI.Controls
+{
+    public class ListView : SimpleControl, IInputListener
+    {
+        public Color TextColor { get; set; } = Color.White;
+        public Color SelectedTextColor { get; set; } = Color.Black;
+
+        private int selectedIndex = -1;
+        private int totalCount = -1;
+
+        public ListView()
+        {
+            Content = new VerticalScrollPanel()
+            {
+                Content = new VerticalStackPanel()
+            };
+        }
+
+        public string SelectedText
+        {
+            get
+            {
+                try
+                {
+                    var selectedItem = ((Content as VerticalScrollPanel).Content as VerticalStackPanel).Children.ElementAt(selectedIndex);
+                    var current = selectedItem as TextBlock;
+                    return current.Text;
+                }
+                catch
+                {
+                    return string.Empty;
+                }
+            }
+        }
+
+        public List<string> Items
+        {
+            set
+            {
+                totalCount = value.Count;
+                selectedIndex = value.Any() ? 0 : -1;
```

Comment:
Instead of using a "magic" value of `-1`, it's better to make the `selectedIndex` field nullable and let the `null` value represent no selection.